scripts/ci/pr-bot/shared/commentStrings.ts

```diff
@@ -39,6 +39,8 @@ export function assignReviewer(labelToReviewerMapping: any): string {
     }
   }
 
+  commentString += "Note: If you would like to opt out of this review, comment `assign to next reviewer`.\n"
```

Comment:
We probably want some extra whitespace before this. Also, could you include it in the multiline string below instead of having multiple appends?